dora/shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -118,17 +120,20 @@ public void generateCapacityReport(GetWorkerReportOptions options, AlluxioConfig
       workerInfoList = mBlockMasterClient.getWorkerReport(options);
     }
     if (workerInfoList.size() == 0) {
-      print("No workers found.");
+      mPrintStream.println("No workers found.");
       return;
     }
     Collections.sort(workerInfoList, new WorkerInfo.LastContactSecComparator());
 
     collectWorkerInfo(workerInfoList);
-    printAggregatedInfo(options);
-    printWorkerInfo(workerInfoList);
+    generateAggregatedInfo(options);
+    generateWorkerInfo(workerInfoList);
     if (workerRegisterToAllMasters) {
       printWorkerAllMasterConnectionInfo(allMastersWorkerInfo);
     }
+    mPrintStream.printf("Capacity information for %s workers:%n",
```

Comment:
hmmm i think we should either print all random strings (line by line) or print purely json. Otherwise the user can't parse the full output as json?

Reply:
Yes, I agree. This command is not used in Golang CLI, so I will revert it.